examples/diffusion.md

```diff
@@ -26,8 +26,8 @@ accelerate launch  \
 	--module align_anything.trainers.text_to_image.sft_diffusion \
 	--model_name_or_path ${MODEL_NAME_OR_PATH} \
 	--train_datasets ${TRAIN_DATASETS} \
-        --train_template DiffusionDB \
-        --train_optional_args large_random_1k \
+    --train_template DiffusionDB \
+    --train_optional_args large_random_1k \
```

Comment:
It appears that there is an issue with the indentation in the scripts located under `examples`. Please fix it.